downloadApolloSchema.sh

```diff
@@ -0,0 +1,9 @@
+./gradlew downloadApolloSchema --endpoint="https://rickandmortyapi.com/graphql" --schema="sample/src/main/graphql/com/chuckerteam/chucker/sample/schema.json"
+
+# GraphQL Schema Update
+# Open Android Studio terminal
+# Run command `./downloadApolloSchema.sh`
+# Format the file (`ctrl+option+i` && `option+cmd+L`), if need
+# Replace `"null"` to `null`, if need
+# Go to the top of the file anc click the run icon "Generate GraphQL SDL schema file"
```

Comment:
```suggestion
# Go to the top of the file and click the run icon "Generate GraphQL SDL schema file"
```

Reply:
Which Icon? Inside which IDE?

Reply:
Is the play icon (will update the name and icon)
▶️

Add the information about IDE supported.